.baseline/checkstyle/checkstyle-suppressions.xml

```diff
@@ -7,6 +7,9 @@
 <!-- IMPORTANT ECLIPSE NOTE: If you change this file, you must restart Eclipse
  for your changes to take effect in its Checkstyle integration. -->
 <suppressions>
+    <!-- Eclipse/Idea APT generated code -->
+    <suppress files="[/\\]generated_src[/\\]" checks="." />
```

Comment:
again, this sounds like something gradle-processors should be doing/does, not baseline. The only reason it's generated_src is because gradle-processors defines it to be so. If you use idea, you should also be excluding generated_testSrc.